hack/ginkgo-e2e.sh

```diff
@@ -169,15 +169,43 @@ elif [[ "${E2E_TEST_DEBUG_TOOL}" == "gdb" ]]; then
   program=("gdb")
 fi
 
+if [[ -n "${E2E_REPORT_DIR:-}" ]]; then
+  GINKGO_JSON_REPORT_FILE+="${E2E_REPORT_DIR}/ginkgo_report.json"
+fi
+
+if [[ -z "${E2E_REPORT_DIR:-}" ]]; then
+  for i in "$@"
+  do
+    if [[ "$i" =~ ^--report-dir* ]]; then
+      IFS=" " read -r -a attr <<< "${i//=/ }" && GINKGO_JSON_REPORT_FILE+="${attr[1]}/ginkgo_report.json"
+      break
+    fi
+  done
+fi
+
+debug_opt=()
+# Here are some options which will be helpful for debugging, for example, test hangs for some reason (https://github.com/kubernetes/kubernetes/issues/111086).
+
+# Unmask this line to circumvent cases where the code grabbing the stdout/stderr pipe
+# is not under the framework control and may cause hangs (https://github.com/onsi/ginkgo/issues/851)
+# this option will also help to reveal more information when the test hangs (https://github.com/onsi/ginkgo/issues/970).
+# debug_opt+=(--ginkgo.output-interceptor-mode=none)
+
+# Unmask this line in order to emits more output including GinkgoWriter contents.
+# debug_opt+=(--ginkgo.v)
```

Comment:
Would that actually change anything?

We already set verbose output in https://github.com/kubernetes/kubernetes/blob/9e65786254c438942ca45d2c447d6318417fc287/test/e2e/framework/test_context.go#L358-L359 so I suspect adding `--ginkgo.v` will not make a difference.

Reply:
yes, will remove this one either.